test/functional/tests/initialize/test_clean_reboot.py

```diff
@@ -29,52 +27,66 @@
 @pytest.mark.require_plugin("power_control")
 def test_load_after_clean_shutdown(reboot_type, cache_mode, filesystem):
     """
-        title: Planned system shutdown test.
-        description: Test for data consistency after clean system shutdown.
-        pass_criteria:
-          - DUT should reboot successfully.
-          - Checksum of file on core device should be the same before and after reboot.
+    title: Planned system shutdown test.
+    description: |
+        Test for data consistency after clean system shutdown.
+    pass_criteria:
+      - DUT reboot successfully.
```

Comment:
Either "DUT rebooted successfully" or "DUT reboot successful" or the old version